documentation/asciidoc/computers/os/updating.adoc

```diff
@@ -120,13 +120,13 @@ sudo apt purge tree
 
 WARNING: Pre-release versions of software are not guaranteed to work. You should not use `rpi-update` on any system unless recommended to do so by a Raspberry Pi engineer. It may leave your system unreliable or even completely broken. It should not be used as part of any regular update process.
 
-The `rpi-update` script is supplied by a third party, "Hexxeh", and also supported by Raspberry Pi engineers. The script source can be found on GitHub at https://github.com/Hexxeh/rpi-update
+The `rpi-update` script was originally written by [Hexxeh](https://github.com/Hexxeh), but is now supported by Raspberry Pi engineers. The script source is in the https://github.com/raspberrypi/rpi-update[rpi-update repository].
```

Comment:
You need to use an AsciiDoc-style link, not a MarkDown-style link for the "Hexxeh" :wink:

Reply:
Fixed. We'll get there eventually 😂.